src/scss/hero.scss

```
@@ -0,0 +1,136 @@
+@import "./mixins.scss";
+.hero {
+  display: flex;
+
+  &__col {
+    width: 50%;
+
+    &_left {
+      position: relative;
+    }
+
+    &_right {
+      @include breakpoint($tab_1248) {
+        padding-left: 2.5rem;
+      }
+
+      @include breakpoint($tab_1024) {
+        width: 100%;
+        padding-left: 0;
+      }
+    }
+  }
+
+  &__image {
+    width: 610px;
+    height: 1000px;
+    display: block;
+
+    img {
+      display: block;
+      width: 100%;
+      height: 100%;
+    }
```

it would be better to avoid such approach. In this case if image will be changed, you can receive really strange picture. 

```suggestion
    display: block;

    img {
      display: block;
      width: 100%;
      max-width: 610px;
    }
```